### PR TITLE
Fix grpc size limitation

### PR DESCRIPTION
**Pull request by Hazhzeng**
## Description

Fixes Issue https://github.com/Azure/azure-functions-python-worker/pull/632/files#diff-9a16b2f8bcbe955d554031aec6e96442R47

The `self._grpc_max_msg_len` is set to `None` due to passing None into `dispatcher.\_\_init\_\_()` method

---

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### Quality of Code and Contribution Guidelines
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-functions-python-worker/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: azure_functions_worker/dispatcher.py [hunk 1]
```diff
@@ -85,10 +85,10 @@ def load_bindings(self):
 
     @classmethod
     async def connect(cls, host, port, worker_id, request_id,
-                      connect_timeout, max_msg_len=None):
+                      connect_timeout):
         loop = asyncio._get_running_loop()
         disp = cls(loop, host, port, worker_id, request_id,
-                   connect_timeout, max_msg_len)
+                   connect_timeout)
```
Comment (vrdmr) [inline azure_functions_worker/dispatcher.py:91]: As you are changing the "max_msg_len", in a subsequent PR, let's start adding type suggestions for the parameters. I'll add a new issue for it.

```
def __init__(self, loop, host, port, worker_id, request_id,
             grpc_connect_timeout, grpc_max_msg_len: int = -1):
```

Comment (TroyWitthoeft) [reply azure_functions_worker/dispatcher.py:91]: Felt this bug in prod.  Was tough to troibleshoot.  How can we tell if our azure app has the latest hotfix bits?

Comment (Hazhzeng) [reply azure_functions_worker/dispatcher.py:91]: Hi @TroyWitthoeft, we finished rollout the hotfix last week. You can check if your function host version is upgraded to 2.0.13998 or 3.0.13978. These two versions should have this HotFix.

Comment (TroyWitthoeft) [reply azure_functions_worker/dispatcher.py:91]: Got it.  We're all set now.  We were able to Kudu/SSH into the box and examine the dispatcher.py to confirm if we had the latest bits.  A delete and redeploy got us the latest bits.  But, I am curious.  We dug around in the portal and couldn't find the functions host version listed anywhere.  Any tips?  Is it on a particular part of the blade? Available over REST?  Printed to logs on startup, etc?  Appreciate it.

**Pull request merged**